getting_started/step_by_step/your_first_game.rst

```diff
@@ -311,13 +401,27 @@ settings - Godot will automatically create a function called
 
 Add this code to the function:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _on_Player_body_entered( body ):
         hide() # Player disappears after being hit
         emit_signal("hit")
         $CollisionShape2D.disabled = true
 
+ .. code-tab:: csharp
+
+    public void _on_Player_body_entered(Godot.Object body)
```

Comment:
I think we don't really need to follow the GDScript naming convention here, but maybe it could make sense in signal callbacks. I'd like to hear others might think about this issue.